### ISSUE TITLE
Can not import component from @react-pdf-viewer/core on IOS 15.x

### ISSUE THREAD

**Issue by quangtd101246** [react-pdf-viewer/react-pdf-viewer#1432]
I can not import {Viewer, Worker} from "@react-pdf-viewer/core" on Chrome or Safari of Iphone IOS version 15.x . I have researched but can not find the solution. How to resolve this issue?

**Comment by phuocng** [comment]
Does it work on your dekstop Chrome with your setup?

**Comment by quangtd101246** [comment]
Yes it work well on windows desktop Chrome. You can use Iphone with IOS version 15.x try this and you try the other view pdf library will be same error. Thanks for reply.

**Comment by quangtd101246** [comment]
Yes it work well on windows desktop Chrome. You can use Iphone with IOS version 15.x try this and you try the other view pdf library will be same error. Thanks for reply.
